rakelib/education_benefits_submission.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+desc 'update all records in education_benefits_submissions table'
+task education_benefits_submission: :environment do
+  # `update_all` is being used because the `vettec` field will reset to `false`
+  # as the form isn't live on staging and prod
+  # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Just curious, what was rubocop yelling at you about?

Reply:
Oohh, `update_all` skips validations. NBD here.

Reply:
Rails/SkipsModelValidations: Avoid using update_all because it skips validations.